src/main/java/ca/corefacility/bioinformatics/irida/service/impl/AnalysisExecutionScheduledTaskImpl.java

```diff
@@ -348,4 +374,28 @@ private void handleJobErrors(AnalysisSubmission analysisSubmission) {
 			return cleanedSubmissions;
 		}
 	}
+
+	/**
+	 * Cleanup any temporary downloaded files and cleanup associated {@link AnalysisSubmissionTempFile} objects
+	 *
+	 * @param submission The analysis submission to clean up temporary downloaded files for
+	 */
+	private void cleanupTemporaryDownloadedFiles(AnalysisSubmission submission) {
+		/*
+		 Cleanup any files that were downloaded from an object store to run an analysis and
+		 remove the analysis submission temp file record from the database.
+		 */
+		if (!storageType.equals(StorageType.LOCAL)) {
```

Comment:
We likely don't need the check here for `StorageType.LOCAL` since they won't have been created if it wasn't local.  Also if we remove this check, we can add a test for the cleanup function.

Reply:
Removed check in eb2d3b4fd2ef1fc91ed99b94fb0574c6bc579ea9